pkgs/tools/misc/czkawka/default.nix

```
@@ -35,7 +35,7 @@ rustPlatform.buildRustPackage rec {
     pango
     gdk-pixbuf
     atk
-    gtk3
+    gtk4
   ];
 
```

according to qarmin/czkawka#790
you could write 

```suggestion

  checkInputs = [
    xvfb-run
  ];

  checkPhase = ''
    runHook preCheck
    xvfb-run cargo test
    runHook postCheck
  '';

```

---

Thanks, is building and running with these changes on my master-czkawka branch.

---

Thanks, it is building fine with these changes. I'll push the updated commit.